src/views/partials/doc-content.ejs

```diff
@@ -151,6 +149,9 @@
   <h1 id="spell-section">Spells</h1>
   <% include ../partials/doc-resource-spells %>
 
+  <h1 id="feats-section">Feats</h1>
```

Comment:
You'll also want to add to the doc-left-col.ejs file so it shows up in the side menu.

Reply:
Done.